src/metrics/spanner-metrics-exporter.ts

```diff
@@ -0,0 +1,138 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {PushMetricExporter, ResourceMetrics} from '@opentelemetry/sdk-metrics';
+import {ExportResult, ExportResultCode} from '@opentelemetry/core';
+import {ExporterOptions} from './external-types';
+import {MetricServiceClient} from '@google-cloud/monitoring';
+import {transformResourceMetricToTimeSeriesArray} from './transform';
+import {status} from '@grpc/grpc-js';
+
+// Stackdriver Monitoring v3 only accepts up to 200 TimeSeries per
+// CreateTimeSeries call.
+const MAX_BATCH_EXPORT_SIZE = 200;
+
+/**
+ * Format and sends metrics information to Google Cloud Monitoring.
+ */
+export class CloudMonitoringMetricsExporter implements PushMetricExporter {
```

Comment:
Do we need to export this class ? This should be used with in the client library.

Reply:
Even within the current project need to export the class to use it in other classes (e.g. `import {CloudMonitoringMetricsExporter} from ...`).